addons/website_sale_delivery/models/sale_order.py

```diff
@@ -124,66 +86,60 @@ def _get_delivery_methods(self, cr, uid, order, context=None):
                 # RIM: hack to remove in master, because available field should not depend on a SOAP call to external shipping provider
                 # The validation error is used in backend to display errors in fedex config, but should fail silently in frontend
                 _logger.debug("%s removed from e-commerce carrier list. %s" % (carrier.name, e))
```

Comment:
Check with @rim-odoo whether that has been fixed and `_get_delivery_method` couldn't just be `return self.env['delivery_carrier'].search([('website_published', '=', True), ('available', '=', True)])`